### DISCUSSION TITLE
Support for Legion 5i 15ITH6H-82JH on Linux

### DISCUSSION THREAD

**Discussion by m-a-z-z-z** [4JX/L5P-Keyboard-RGB#113] [Ideas]
I tried this app on windows and it works, but it doesn't work on Linux. Any chance support could be added or could you point me towards where this support can be added in the code? Cheers

**Comment by 4JX** [comment]
If it works on windows it should detect it just fine on Linux as well, have you checked out the [usage section](https://github.com/4JX/L5P-Keyboard-RGB#usage)?

**Reply by m-a-z-z-z** [reply]
My bad, ran it with root privileges but didn't have the 99-kblight.rules file. Added it and works fine, thanks for the quick response!
